api/app/command.py

```diff
@@ -272,6 +272,24 @@ def get_vulns(
     if package_manager:
         filters.append(models.Dependency.package_manager == package_manager)
 
+    count_query = select(func.count(models.Vuln.vuln_id.distinct()))
+    count_query = count_query.join(
+        models.Affect, models.Vuln.vuln_id == models.Affect.vuln_id
```

Comment:
```suggestion
        models.Affect
```
外部キーがあるため、結合条件不要。173行目と合わせましょう。

Reply:
修正しました。